sdk/python/README.md

```diff
@@ -87,111 +78,29 @@ Now that you have completed all of the above steps, you have a working provider
 You now have:
 
 1. A `provider/` folder containing the building and implementation logic
-   1. `cmd/`
-      1. `pulumi-gen-xyz/` - generates language SDKs from the schema
-      2. `pulumi-resource-xyz/` - holds the package schema, injects the package version, and starts the gRPC server
-   2. `pkg`
-      1. `provider` - holds the gRPC methods (and for now, the sample implementation logic) required by the Pulumi engine
-      2. `version` - semver package to be consumed by build processes
+    1. `cmd/pulumi-resource-xyz/main.go` - holds the provider's sample implementation logic.
 2. `deployment-templates` - a set of files to help you around deployment and publication
 3. `sdk` - holds the generated code libraries created by `pulumi-gen-xyz/main.go`
 4. `examples` a folder of Pulumi programs to try locally and/or use in CI.
 5. A `Makefile` and this `README`.
 
-### Writing the schema
```

Comment:
This is redundant now I'm guessing?

Reply:
I have not gotten this far into the intricacies of provider development and do not have an opinion on this yet.

Can you explain your thinking with more context?

Reply:
Just was wondering why you PR removes this "Writing the schema" section.

Reply:
Did that not change automatically from templating things?

I did not remove it intentionally. If it comes from templating which changed then I could benefit from guidance on how to re-place it.

Reply:
It should be unnecessary to write the schema manually, since this is a pulumi-go-provider based provider.